test/unit/lib/plugins/aws/provider.test.js

```diff
@@ -690,6 +428,228 @@ describe('AwsProvider', () => {
 });
 
 describe('test/unit/lib/plugins/aws/provider.test.js', () => {
+  describe('#getCredentials()', () => {
+    before(async () => {
+      // create default aws credentials file in before so that grouped run can use it
+      await fs.ensureDir(path.resolve(os.homedir(), './.aws'));
+      await fs.outputFile(
+        path.resolve(os.homedir(), './.aws/credentials'),
+        `
+[default]
+aws_access_key_id = DEFAULTKEYID
+aws_secret_access_key = DEFAULTSECRET
+
+[notDefault]
+aws_access_key_id = NOTDEFAULTKEYID
+aws_secret_access_key = NOTDEFAULTSECRET
+
+[notDefaultWithRole]
+source_profile = notDefault
+role_arn = NOTDEFAULTWITHROLEROLE
+`,
+        { flag: 'w+' }
+      );
+    });
+
+    after(async () => {
```

Comment:
This is not needed as the mocked homedir will be cleaned after each test

Reply:
I was not sure :+1: thanks :bow: